### ISSUE TITLE
Spinner custom colors

### ISSUE THREAD

**Issue by vcmorini** [facultyai/dash-bootstrap-components#352]
Hello
I think it would be great to be able to set any possible color to the spinner. Example use-case: branding purposes.
Right now it has predefined color options.
Reference: [link](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/spinner/)
Cheers!

**Comment by tcbegley** [comment]
Hey @vcmorini,

Thanks for suggesting this, we actually do something like this already for `Navbar` and `NavbarSimple`, so it should be pretty straightforward to do the same thing for spinner. I'll get that added.

In the meantime, you can set this with the `spinner_style` prop:

```python
import dash
import dash_bootstrap_components as dbc

app = dash.Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])

app.layout = dbc.Spinner(spinner_style={"color": "green"})

if __name__ == "__main__":
    app.run_server(debug=True)
```

**Comment by tcbegley** [comment]
I've addressed this in #366, will be available in the next release.
